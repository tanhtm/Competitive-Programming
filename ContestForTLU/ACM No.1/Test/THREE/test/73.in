638367202000000000
