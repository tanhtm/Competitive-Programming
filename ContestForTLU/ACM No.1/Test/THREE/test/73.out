2 3 6 7 14 17 18 19 20 23 24 26 27 28 29 30 31 34 
0 5 11 13 21 25 33 35 36 37 
